### PR TITLE
(0.33) Improve error message when loading an invalid class version

### PR DESCRIPTION
**Pull request by thallium**
Port of https://github.com/eclipse-openj9/openj9/pull/15084 for the 0.33 release.

Signed-off-by: Gengchen Tuo <gengchen.tuo@ibm.com>

### CODE REVIEW

File: runtime/nls/cfre/cfrerr.nls [hunk 1]
```diff
@@ -1556,115 +1402,190 @@ J9NLS_CFR_ERR_SEALED_CLASS_HAS_INVALID_NUMBER_SUBCLASSES=
...
+J9NLS_CFR_ERR_PREVIEW_VERSION2.sample_input_1=60
+J9NLS_CFR_ERR_PREVIEW_VERSION2.sample_input_2=65535
+J9NLS_CFR_ERR_PREVIEW_VERSION2.sample_input_3=4
+J9NLS_CFR_ERR_PREVIEW_VERSION2.sample_input_4=main
+J9NLS_CFR_ERR_PREVIEW_VERSION2.sample_input_5=61
+J9NLS_CFR_ERR_PREVIEW_VERSION2.sample_input_6=6
+J9NLS_CFR_ERR_PREVIEW_VERSION2.explanation=Classfiles with minor version 65535 must have the same major version as the VM runtime version and preview must be enabled
+J9NLS_CFR_ERR_PREVIEW_VERSION2.system_action=The JVM will throw a verification or classloading related exception.
+J9NLS_CFR_ERR_PREVIEW_VERSION2.user_response=Run with the correct Java runtime version and use --enable-preview, or re-compile the code with the same version of the runtime.
+# END NON-TRANSLATABLE
```
Comment (pshipton) [inline runtime/nls/cfre/cfrerr.nls:1591]: There is a missing newline here.

**Comment by pshipton** [comment]
> Am I doing it right?

The missing newline indicates you didn't use the git cherry pick operation. Using cherry pick is an easy way to move changes between different branches without missing anything, assuming there are no conflicts.

**Comment by pshipton** [comment]
@keithc-ca fyi

File: runtime/nls/cfre/cfrerr.nls (hunk 1 above)
Comment (keithc-ca) [reply runtime/nls/cfre/cfrerr.nls:1591]: This includes the nls changes from 13a8dfb3313febcfec15af68e612eb3fffdd2824 but not 5c066f985254d94095b8a47e305870e9bd0b4e49. It's probably best to include all the content from the master branch in this file.

Comment (pshipton) [reply runtime/nls/cfre/cfrerr.nls:1591]: Created https://github.com/eclipse-openj9/openj9/pull/15187

Comment (keithc-ca) [reply runtime/nls/cfre/cfrerr.nls:1591]: I think we'll be good after that missing newline is added.

Comment (thallium) [reply runtime/nls/cfre/cfrerr.nls]: > There is a missing newline here.

Should be fixed now

**Review by keithc-ca — APPROVED** [review]

**Review by pshipton — APPROVED** [review]

**Pull request merged**